service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/request/EhrExtractAckHandlerTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.nhs.adaptors.gp2gp.ehr.request;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.xml.sax.SAXException;
+
+import uk.nhs.adaptors.gp2gp.common.service.XPathService;
+import uk.nhs.adaptors.gp2gp.mhs.InvalidInboundMessageException;
+
+@ExtendWith(MockitoExtension.class)
+public class EhrExtractAckHandlerTest {
+
+    @Spy
+    private XPathService xPathService;
```

Comment:
Since we already have integration tests covering external interactions with this class, we could use `@mock` here